vim_onenote.ahk

```diff
@@ -117,77 +216,77 @@ Send, {End}{Enter}
 Gosub InsertMode
 return
 
+r::
+    send +{right}
+    gosub, InsertMode
+    ; Wait for single key to be pressed, sends that key and returns to normal
+    input, inp, V E L1
+    send {left}
+    gosub, NormalMode
+return
+
 ; undo
 u::Send, ^z
-return 
 ; redo.
 ^r::Send, ^y
-return 
 
-+G:: Send, ^{End}
 ; G goto to end of document
-return
+sG(){
```

Comment:
Context free: You spend MUCH more time reading code then writing it -- optimize for readers. See other comments on the value of having this explicit